spec/controllers/v0/benefits_claims_controller_spec.rb

```diff
@@ -221,10 +221,10 @@
             get(:show, params: { id: '600383363' })
           end
           parsed_body = JSON.parse(response.body)
-          expect(parsed_body.dig('data', 'attributes', 'trackedItems').size).to eq(13)
+          expect(parsed_body.dig('data', 'attributes', 'trackedItems').size).to eq(15)
```

Comment:
@iandonovan Are these tests that check the # of tracked items going to be flaky? Maybe we could check attributes of the new tracked items.

Reply:
They'll be flaky in that they'll need to be updated alongside the cassette, but the counts will be the same each time the test runs. I do agree that these tests are kind of a mess, though. I'll take a step back and implement something cleaner.